app/models/airtable_project.rb

```diff
@@ -2,21 +2,27 @@
 
 
 class AirtableChannelList < Airrecord::Table
-  self.base_key = "appUZignaw9kIHqTR"
+  self.base_key = "appwE2GxYI3JzuGix"
```

Comment:
Should these be moved to $CONFIG?

Reply:
yes i will put them in environmental variables good call